### PR TITLE
Migration guide 5to6

### PR DESCRIPTION
**Pull request by danizavtz**
Hello did a pull request for increment the documentation, did a migration guide from previous version from 5 to 6. As a contribution to address this issue: [https://github.com/express-validator/express-validator/issues/735](url)
Can you guide me to improve this pull request to be torn itself a migration guide `de facto` from express validation? I used the documentation already existing to the current documentation, but i have some legacy projects that use express validator and I can use it as a "real project" migration guide.
I know express is an unopinionated framework, so i did a minimal documentation guide for not suppose an architecture.

### CODE REVIEW

File: docs/migration-guide.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+app.use(bodyParser.urlencoded({ extended: true
+}));
+
+app.use(require('./server/index'));
+.
+.
+.
+module.exports = app;
+```
+Now in your route handler we should import `express-validator` module (line 3).
```
Comment (fedeci) [inline docs/migration-guide.md:77]: I think this should be: "in our root handler"

File: docs/migration-guide.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
+---
+id: migration
+title: Migration guide
+---
+This tutorial guide your throught to one way to migrate your earlier version of express-validator to version 6.
```
Comment (fedeci) [inline docs/migration-guide.md:5]: You mean "through", don't you?

**Review by fedeci — COMMENTED** [review]
This looks good to me👍🏻.

File: docs/migration-guide.md (hunk 1 above)
Comment (danizavtz) [reply docs/migration-guide.md]: Thanks for detailed feedback. Just updated the documentation.

File: docs/migration-guide.md (hunk 2 above)
Comment (danizavtz) [reply docs/migration-guide.md]: Thanks for detailed feedback. Just updated the documentation.

**Comment by gustavohenke** [comment]
Closing since #948 went in first after all..

**Pull request closed without merging**